catroid/src/main/res/values/strings.xml

```diff
@@ -1079,6 +1079,9 @@
     <string name="brick_default_embroidery_file">embroidery.dst</string>
     <!--  -->
 
+    <!-- Device bricks -->
+    <string name="brick_reset_timer">Reset Timer</string>
```

Comment:
```suggestion
    <string name="brick_reset_timer">Reset timer</string>
```
All our bricks use normal English capitalization rules. Also, see the jira ticket.